app/api/search/specs/search.spec.js

```diff
@@ -119,6 +119,25 @@ describe('search', () => {
         });
       });
 
+      describe('when quotes are present', () => {
+        it('should not fail if they are paired', done => {
+          search
```

Comment:
@fnocetti pls change the specs so they use async/await, also it is always a good idea to expect something. based on jest [docs](https://jestjs.io/docs/en/asynchronous#resolves-rejects) this should work and fail when there is a rejection.

```
await expect(
  search.searchSnippets('"text"', ids.batmanFinishes, 'es')
).resolves.toBeDefined();